parsl/providers/kubernetes/kube.py

```diff
@@ -63,6 +75,10 @@ def __init__(self,
                  init_blocks: int = 4,
                  min_blocks: int = 0,
                  max_blocks: int = 10,
+                 max_cpu: float = 2,  # in cpu units
+                 max_mem: str = "500Mi",  # in Mi, Gi
```

Comment:
These units do not need to be documented here - use the style used elsewhere, where the units are in the main docstring

Reply:
fixed the comments and docstrings.